benchmarks/project/src/main/java/io/zeebe/Worker.java

```diff
@@ -62,12 +64,13 @@ public void run() {
                   } else {
                     try {
                       Thread.sleep(completionDelay);
-                    } catch (Exception e) {
+                    } catch (final Exception e) {
                       e.printStackTrace();
                     }
                     requestFutures.add(command.send());
                   }
                 })
+            .backoffSupplier(newBackoffBuilder().minDelay(250L).backoffFactor(2).build())
```

Comment:
Any reason to not use the defaults?

Reply:
I wrote this commit message:
```
chore(benchmark): configure backoff for workers 

To improve the benchmark, it makes sense to reduce the number of
unnecessary ActivateJobs requests the workers send. This should provide
more resources to actually completing jobs and creating new workflow
instances. The default minDelay of 50ms, is too small for this case
and can easily be increased for our benchmarks.

Even if a request receives a RESOURCE_EXHAUSTED error, the delayed
request had about 50% change that it succeeds afterwards (we actually
want to optimise to increase that chance). So it never get's a real
chance to build up the delay (exponentially). For the benchmark, I
therefore increased the min delay to 250ms and a faster increase at 2
times. Expected intervals are thus: 250ms, 500ms, 1s, 2s, 4s and 5s.

These values seem to reduce the number of ActivateJobs requests by half
compared to the defaults without stressing the workers with too many
jobs.

This might not hold for real world applications.
```

WDYT? Are these sensible values? Should we also change the defaults?